torch/_inductor/comms.py

```diff
@@ -196,206 +242,6 @@ def estimate_op_runtime(snode: BaseSchedulerNode) -> float:
     return runtime
 
 
-def compute_node_users(
```

Comment:
We can delete all this code since now, `raise_comms/sink_waits` use a very similar logic to this? Deleting code is great :)

Reply:
Yes. To make raise_comms/sink_waits more effective, it used similar logic as reorder_compute_for_overlap (i.e., ready queue based scheduling).